public/app/features/dashboard/components/ShareModal/SharePublicDashboard.test.tsx

```diff
@@ -88,6 +90,13 @@ describe('SharePublic', () => {
 
     fireEvent.click(screen.getByText('Public Dashboard'));
 
-    await waitFor(() => screen.getByText('Enabled'));
+    await screen.findByText('Welcome to Grafana public dashboards alpha!');
   });
+
+  // test when checkboxes show up
```

Comment:
@owensmallwood @pianohacker would be cool to get these test stubs done before we merge. Can we split these up?